tests/providers/test_user_agent.py

```diff
@@ -19,3 +20,8 @@ def test_ios_platform_token(self, faker, num_samples):
             match = self.ios_token_pattern.fullmatch(faker.ios_platform_token())
             assert match.group('apple_device') in UaProvider.apple_devices
             assert match.group('ios_version').replace('_', '.') in UaProvider.ios_versions
+    
```

Comment:
[According to the linter](https://github.com/joke2k/faker/pull/1509/checks?check_run_id=3610305718#step:5:15), this line contains some spaces. Could you remove them?

Reply:
Updated.